src/index.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import { render } from "react-dom";
+import App from "./App";
+import * as serviceWorker from "./serviceWorker";
+
+render(<App />,document.getElementById("root"));
+
+// If you want your app to work offline and load faster, you can change
+// unregister() to register() below. Note this comes with some pitfalls.
+// Learn more about service workers: https://bit.ly/CRA-PWA
+serviceWorker.register();
```

Comment:
What pitfalls? Also the link is outdated.

When something comes with pitfalls, it's always good to see a comment covering those. I.e. something like

> Service Workers can lead to x, which is acceptable since this project does y

Reply:
I will remove that was a create-react-app boiler plate.